### PR TITLE
Test fetching of specs from github raw

### PR DESCRIPTION
**Pull request by wirednkod**

### CODE REVIEW

**Comment by wirednkod** [comment]
No tests can run for this PR due to the:
```
: Timeout - Async callback was not invoked within the 30000 ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 30000 ms timeout specified by jest.setTimeout.Error:

      19 |   }, 30000);
```

File: packages/connect/downloadSpecs.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import http from 'https';
+import fs from 'fs';
+
+const commonBridgesRepo = 'https://raw.githubusercontent.com/paritytech/smoldot/main/bin';
```
Comment (raoulmillais) [inline packages/connect/downloadSpecs.js:4]: Bridges?  I think you forgot which team you're on today :stuck_out_tongue:

File: packages/connect/src/Detector.test.ts [hunk 2]
```diff
@@ -35,9 +33,8 @@ describe('Initialize Detector without extension', () => {
   // DEACTIVATED for now due to chainSpecs - once Kusama specs are available this should be activated
```
Comment (raoulmillais) [inline packages/connect/src/Detector.test.ts:33]: I'd prefer if you used `test.skip` then the syntax continues to be built by typescript.  Keep the DEACTIVATED comment though

**Review by raoulmillais — CHANGES_REQUESTED** [review]

File: packages/connect/downloadSpecs.js (hunk 1 above)
Comment (wirednkod) [reply packages/connect/downloadSpecs.js:4]: Got this code from bridges team :) Its an honest mistake

**Review by raoulmillais — APPROVED** [review]

**Pull request merged**